src/fr24/tui/tui.py

```diff
@@ -70,7 +70,7 @@ class FR24(App[None]):
 
     def compose(self) -> ComposeResult:
         self.auth: Authentication | None = None
-        self.client = httpx.AsyncClient()
+        self.client = httpx.AsyncClient(http2=True)
```

Comment:
I'm not sure why you need that option?

Reply:
This traces back to this issue: https://github.com/cathaypacific8747/fr24/issues/23#issuecomment-2125624974 - fr24's servers sometimes returns 464 errors if HTTP/1 is used. HTTP/2 was enabled by default to keep it consistent with requests made by the browser

Reply:
OK all clear, would deserve a comment in the code 😉